readme.md

```diff
@@ -82,20 +86,39 @@ require('@zeit/ncc')('/path/to/input', {
   v8cache: false, // default
   quiet: false, // default
   debugLog = false // default
-}).then(({ code, map, assets }) => {
-  console.log(code);
-  // Assets is an object of asset file names to { source, permissions, symlinks }
-  // expected relative to the output code (if any)
+}).then(({ files, symlinks }) => {
+  // files: an object of file names to { source, permissions }
+  // symlinks: an object of symlink mappings required for the build
+  // The main build file is located at files[filename]:
+  files['build.js'].source
 })
 ```
 
+Multiple entry points can be built by providing an object to build. In this case, those files are avaiable as additional output files:
+
+```js
+require('@zeit/ncc')({
```

Comment:
In the next PR when we breakout `build` vs `watch`, I think it might be a good idea to make this a separate function for multiple entry points such as `buildMany`. This way we can avoid function overloading. Or perhaps `build` should accept an array of `filenames` instead?

Reply:
Note that this argument is `entry`, and `filename` is the Webpack `output.filename`, so that we are completely symmetrical with Webpack. I think keeping this as a single function makes sense for that reason.

Reply:
For the single function approach, what do you think about changing line 74 above to be an array?

Reply:
We could support the array form too.

The `string` for entry is allowed by Webpack, so I don't see why we shouldn't.

Reply:
So how about `filename: string | string[]` and that can be passed to webpack. Then we don't need the object usage you defined here because the array will be paths.

Instead of this:

```js
{
  entry1: '/path/to/input1',
  entry2: '/path/to/input2
}
```

Use this:

```js
{
   filename: ['/path/to/input1', '/path/to/input2']
}
```

Reply:
The thing is we are naming the entry points, too just like with Webpack (https://webpack.js.org/configuration/entry-context#entry), where the filename is rule-based (`filename: '[name].js'`).

Reply:
I was thinking that the name could match the file path. Is that going to cause a problem?

Reply:
I've pushed up a commit to support the array form as well.

Reply:
I'm ok with the object case but I think we need to use either objects every time or arrays every time. I don't like that the multi-entry-point takes 2 parameters: object of entry points and then a file name. Maybe I read the example incorrectly.

Reply:
The filename is the output filename, just like with Webpack.

Reply:
@guybedford I think we should minimize the amount of Webpack syntax that leaks through to the `ncc` API. It seems like `'[name].js'` is webpack syntax.

I looked through the code again and I think I got confused. The Array part was misleading and can be removed.

My preferred solution (now that I understand this code a little better) is to change line 72 so that it is no longer using a string. Instead it should use an Object to map the file name to the file path. Alternatively, I would be okay with using an Array here instead of an Object so that file names are the same as file paths.

Then the `filename` can be removed completely.

I would be happy to jump on a call if you need to discuss further.

Reply:
Ok I've pushed up the change to deprecate `filename`, and instead rely entirely on the mappings from the entry object for output naming. I've also included automatic extension adding to this for convenience.